tests/infra/network.py

```diff
@@ -1530,13 +1530,10 @@ def wait_for_snapshots_to_be_committed(src_dir, list_src_dir_func, timeout=20):
                     )
                     return False
 
-                with node.client(self.consortium.get_any_active_member().local_id) as c:
-                    logs = []
-                    for _ in range(self.args.snapshot_tx_interval // 2):
-                        r = c.post("/gov/ack/update_state_digest", log_capture=logs)
-                        assert (
-                            r.status_code == http.HTTPStatus.OK.value
-                        ), f"Error ack/update_state_digest: {r}"
+                # Update state digest as a neutral write operation, to advance commit
+                member = self.consortium.get_any_active_member()
+                r = member.update_ack_state_digest(node)
```

Comment:
That only seems to do a single write, where the code was previously adding half a snapshot tx interval worth, to make sure a snapshot was triggered. Does the test that make use of that still work? If so, it probably needs looking at.

Reply:
I've restored the multiple writes.

I _think_ that missing this will introduce an occasional failure in any test doing recoveries - _sometimes_ we will get an error from trying to use an uncommitted snapshot. However, I wasn't able to repro that in a local loop of `recovery` tests.

Reply:
Let's ticket this for further investigation.